src/expr/proof_checker.h

```diff
@@ -185,15 +187,17 @@ class ProofChecker
   uint32_t d_pclevel;
   /**
    * Check internal. This is used by check and checkDebug above. It writes
-   * checking errors on out. We treat trusted checkers (nullptr in the range
-   * of the map d_checker) as failures if useTrustedChecker = false.
+   * checking errors on out when enableOutput is true. We treat trusted checkers
+   * (nullptr in the range of the map d_checker) as failures if
+   * useTrustedChecker = false.
```

Comment:
Spent a couple minutes confused again by "trusted" meaning that we are trusting something untrustworthy. :)